biogeophys/EDSurfaceAlbedoMod.F90

```diff
@@ -89,6 +89,7 @@ subroutine ED_Norman_Radiation (nsites, sites, bc_in, bc_out )
            ifp = 0
            currentpatch => sites(s)%oldest_patch
            do while (associated(currentpatch))  
+             if(currentpatch%nocomp_pft_label.ne.0)then
```

Comment:
added this:
              ! do not do albedo calculations for bare ground patch in SP mode                                           
      	      ! and (more impotantly) do not iterate ifp or it will mess up the indexing wherein                         
      	      ! ifp=1 is the first vegetated patch.